src/main/resources/i18n/strings.properties

```diff
@@ -321,6 +321,14 @@ preferences.updates.currentVersion=Current Version: %s
 preferences.updates.autoUpdateCheck=Check for updates automatically
 preferences.updates.checkNowBtn=Check Now
 preferences.updates.updateAvailable=Update to version %s available.
+preferences.updates.lastUpdateCheck=Last check: %s
+preferences.updates.lastUpdateCheck.never=never
+preferences.updates.lastUpdateCheck.recently=recently
+preferences.updates.lastUpdateCheck.daysAgo=%s days ago
+preferences.updates.lastUpdateCheck.hoursAgo=%s hours ago
+preferences.updates.checkFailed=Looking for updates failed. Please check your internet connection and try again later.
```

Comment:
"Please check your internet connection **or** try again later."

In any case, it is strange that we hint to a solution, but show the log file location anyway. Non-technical users might have no idea what to do with this.